docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex) and [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Publishing
+
+Blazor apps are published for deployment in Release configuration with the [dotnet publish](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish) command:
```

Comment:
I think we need to clarify a bit what "publish" means in this context. Publishing in the .NET Core world means something pretty specific that I believe is different from what most people think of as publishing. Publishing for us means preparing the app to be deployed. You can't just deploy the built output.

So maybe say something like: "To prepare a Blazor app for deployment it first needs to be *published*. Publishing the app packs the application and its dependencies into a folder for deployment to a hosting system. Blazor apps are published for deployment using the [dotnet publish]((https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish) command. . ."

Reply:
I have an idea that given how much I just ripped the guts out of this that I should try to work that back in on the following commit (after the next one). I'm performing a big overhaul of the layout here. Let's see where "publish" fits after the next commit, and then I can roll that language into it. 

I think that in an IDE one can build/publish and probably (?) get to the assets (*dist* folder) without running the `dotnet publish` command directly. Therefore, let's see where the draft goes on the next commit and where it fits.